packages/apps/transfer/src/pages/faucet/existing/notification.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+  Button,
+  INotificationProps,
+  Notification,
+  NotificationBody,
+  NotificationFooter,
+  styled,
+  SystemIcons,
+} from '@kadena/react-components';
+
+import { RequestStatus } from '.';
+
+import React, {
+  FC,
+  MouseEventHandler,
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+
+type FormNotificationProps = Pick<INotificationProps, 'title'> & {
```

Comment:
move styles in a different file please, by using ts as its extension it will not get built during next built

Reply:
You mean move `type`s to a different file?

Reply:
yeah but only if they are re-usable
if they're not used in any other file then just ignore what I said

Reply:
For now it isn't, if it will then I will refactor 👍